src/core/toga/constants/color.py

```diff
@@ -0,0 +1,14 @@
+RED = 'RED'
+GREEN = 'GREEN'
+BLUE = 'BLUE'
+BLACK = 'BLACK'
+WHITE = 'WHITE'
+BROWN = 'BROWN'
+CYAN = 'CYAN'
+MAGENTA = 'MAGENTA'
+YELLOW = 'YELLOW'
+ORANGE = 'ORANGE'
+PURPLE = 'PURPLE'
+LIGHTGRAY = 'LIGHTGRAY'
+DARKGRAY = 'DARKGRAY'
+GRAY = 'GRAY'
```

Comment:
Where style is involved, I'd prefer to take the lead of CSS - so colors identifiers would be `LightGray`, rather than `LIGHTGRAY`